orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -730,14 +733,25 @@ Function Get-AuditStorageInformation {
                 }
         }
         # Let's get audit local storage information
-        elseif ($ToolkitConfigurationJson.Configuration.Audit -and
-                $ToolkitConfigurationJson.Configuration.Audit.StorageType.ToLower() -eq "local") {
+        elseif (($ToolkitConfigurationJson.Configuration.Audit -and
+                $ToolkitConfigurationJson.Configuration.Audit.StorageType.ToLower() -eq "local") -or
+                $null -ne $ToolkitConfigurationJson.Configuration -or 
```

Comment:
add $null -ne $ToolkitConfigurationJson

Reply:
Added the condition for $ToolkitConfigurationJson on Audit and Cache.